README.md

```diff
@@ -1,2 +1,66 @@
-# reggaemedia
-the reggaemedia website
+# Getting started
```

Comment:
let's start in a bit other way, something like this:


# Getting started

1. Install dependencies

> yarn

2. Create the `.env` file in the root directory consists of following variables:

- `AMAZON_ACCESS_KEY`
- `AMAZON_SECRET_KEY`
- `AMAZON_REGION`

---

- `BUCKET` — static assets s3 name
- `TABLE_ARTICLES` — dynamodb articles table
- `TABLE_SETTINGS` — dynamodb settings table

---

- `COGNITO_CLIENT_ID`
- `COGNITO_CLIENT_SECRET`
- `COGNITO_USER_POOL_ID`
- `COGNITO_CALLBACK_URL`

---

- `NEXTAUTH_SECRET`
- `NEXTAUTH_URL`

---

- `ADMIN_EMAIL` — contact form submission destination email


> All necesseray infrastructure can be created and deployed using the `yarn stack:deploy` command. Only following parameters are required for this step: `AMAZON_ACCESS_KEY`, `AMAZON_SECRET_KEY`, `AMAZON_REGION` and `COGNITO_CALLBACK_URL`.